packages/web/src/components/banner/AppBannerWrapper.tsx

```diff
@@ -0,0 +1,15 @@
+import cn from 'classnames'
+
+import { isMobile } from 'utils/clientUtil'
+
+import styles from './AppBannerWrapper.module.css'
+
+export const AppBannerWrapper = ({
```

Comment:
Since we don't have any banners on mobile web, do we need this component? Could we just put these styles on the base banner CTA? I wonder if we shouldn't support this at all

Reply:
Yeah this is a gross leftover remnant from prototyping/testing, kinda want to leave it so at least we have the option of banners on mobile web? The _real_ goal was making it completely separate (ie not take children) and have the banner list self contained inside of it, and have it decide which to show but that proved to hard to separate from the required update logic cleanly

Reply:
Yeah makes sense. Don't mind leaving it in